daemon/src/taker_cfd.rs

```diff
@@ -167,6 +185,65 @@ impl<O, M, W> Actor<O, M, W> {
 
         Ok(())
     }
+
+    async fn handle_auto_rollover(&mut self) -> Result<()> {
+        tracing::debug!("Auto rollover");
+
+        let mut conn = self.db.acquire().await?;
+        let cfds = load_all_cfds(&mut conn).await?;
+
+        // cleanup all pending proposals because auto-rollover will create a new one
+        self.current_pending_proposals.clear();
+
+        // Only cfds in state `Open` are allowed for auto-rollover proposal
+        for cfd in cfds
+            .iter()
+            .filter(|cfd| matches!(cfd.state, CfdState::Open { .. }))
+        {
+            let now = OffsetDateTime::now_utc();
+
+            let elapsed_since_creation =
```

Comment:
I think this variable name is now stale.

Reply:
do you mean that it should be renamed? how about `elapsed_since_open`

Reply:
You are using `cfd.expiry_timestamp()` yeah? So this computation is no longer based on the _creation_ timestamp.